community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -201,9 +203,10 @@ final void forceClose()
         }
     }
 
-    final String authSubjectName()
+    final String username()
     {
-        return transaction.mode().name();
+        AccessMode mode = transaction.mode();
```

Comment:
i find it strange that kernel statement perform this dance, when we will have mode not an instance of AuthSubject?